extensions/noteplan-3/package.json

```
@@ -28,6 +29,16 @@
       "mode": "view"
     }
   ],
+  "preferences": [
+    {
+      "name": "extension",
+      "type": "textfield",
+      "required": false,
+      "title": "Note extension",
+      "default": "txt",
+      "description": "The extension you use for notes"
```

Let's try something more explicit and have similar naming for the title and name, like
```suggestion
      "name": "fileExtension",
      "type": "textfield",
      "required": false,
      "title": "File Extension",
      "default": "txt",
      "description": "The extension you use for notes"
```

---

👍